optimum/exporters/tasks.py

```diff
@@ -1718,38 +1702,18 @@ def standardize_model_attributes(
             library_name (`Optional[str]`, *optional*)::
                 The library name of the model. Can be any of "transformers", "timm", "diffusers", "sentence_transformers".
         """
-        library_name = TasksManager.infer_library_from_model(
-            model_name_or_path, subfolder, revision, cache_dir, library_name
-        )
```

Comment:
You can not remove it with library_name hinted as `library_name: Optional[str] = None`. You can replace that by a `TasksManager._infer_library_from_model` call.

Reply:
No. `library_name` is hinted as `Optional[str]`.

Reply:
replaced by TasksManager._infer_library_from_model